src/makeCalculator.js

```diff
@@ -3,8 +3,37 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(operation, x) {
```

Comment:
[NAMING] The `makeCalculator` function should not take any parameters according to the task description. It should return a calculator object with predefined methods and properties.